cli/src/build-settings.ts

```diff
@@ -13,35 +14,46 @@ function isStringDict(x: JSON.Object): x is Dict<string | null> {
 
 export default class BuildSettings {
   private rustc: string;
+  private nodeVersion: string | null;
   private env: Dict<string | null>;
 
-  constructor(rustc: string, env: Dict<string | null>) {
+  constructor(rustc: string, nodeVersion: string | null, env: Dict<string | null>) {
     this.rustc = rustc;
+    this.nodeVersion = nodeVersion
```

Comment:
Missing semicolon. (We should adopt a linter and autoformatter! Not in this PR though.)